src/components/EditorsColumn.jsx

```diff
@@ -39,7 +39,7 @@ export default function EditorsColumn({
 
   const editors = [];
 
-  visibleLanguages.forEach(({language, index}) => {
+  visibleLanguages.forEach(({language, index}, idx) => {
```

Comment:
Can we rename the `index` and `idx` variables to make it clearer what each of them is indexing?

Reply:
@outoftime thanks for the quick turnaround. Before I put a new change up, what do you think about renaming like this?

`idx => visibleLanguagesIndex`

`index => languageIndex`

Reply:
Yep sounds good to me!

Reply:
@outoftime I've update the index names.  I also ran prettier, which should account for all the diffs in the file besides the renaming.